src/main/java/org/nantipov/utils/wordhugopress/config/job/convertposts/ConvertPostsReaderConfig.java

```diff
@@ -51,44 +50,16 @@ public ConvertPostsReaderConfig(SourcesSettings sourcesSettings) {
     }
 
     private ItemReader<Post> reader(String sourceName, SourcesSettings.Source source) {
-        if (source.getTags() != null && !source.getTags().isEmpty())
-            TABLE_PREFIX = source.getWordpressTablePrefix();
-        else
-            TABLE_PREFIX = "wp";
-        
-        SQL_QUERY_POSTS = "SELECT\n" +
-            "    p.post_date, p.ID, p.post_modified, p.post_title, p.post_content, p.post_status,\n" +
-            "    u.display_name user_displayname,\n" +
-            "    tax.taxonomy, tax.term_value,\n" +
-            "    thumbnail.thumbnail_data\n" +
-            "FROM\n" +
-            "    " + TABLE_PREFIX + "_posts p\n" +
-            "    JOIN " + TABLE_PREFIX + "_users u ON (u.ID = p.post_author)\n" +
-            "    LEFT JOIN (\n" +
-            "        SELECT termtax.taxonomy, term.name term_value, tr.object_id post_id\n" +
-            "        FROM " + TABLE_PREFIX + "_term_relationships tr, " + TABLE_PREFIX + "_term_taxonomy termtax, " + TABLE_PREFIX + "_terms term\n" +
-            "        WHERE\n" +
-            "                termtax.term_taxonomy_id = tr.term_taxonomy_id\n" +
-            "                AND term.term_id = termtax.term_id\n" +
-            "        ) tax ON (tax.post_id = p.ID)\n" +
-            "    LEFT JOIN (\n" +
-            "        SELECT pm2.meta_value as thumbnail_data, pm1.post_id\n" +
-            "        FROM " + TABLE_PREFIX + "_postmeta pm1, " + TABLE_PREFIX + "_postmeta pm2\n" +
-            "        WHERE\n" +
-            "                pm1.meta_key = '_thumbnail_id'\n" +
-            "                AND pm2.post_id = pm1.meta_value\n" +
-            "                AND pm2.meta_key = '_wp_attachment_metadata'\n" +
-            "        ) thumbnail ON (thumbnail.post_id = p.ID)\n" +
-            "\n" +
-            "WHERE\n" +
-            "    p.post_type = 'post'\n" +
-            "ORDER BY p.ID";
+        if (source.getWordpressTablePrefix() != null && !source.getWordpressTablePrefix().isEmpty() ){
```

Comment:
I think we should not even check the value at the moment, we can just grab what we have. Otherwise, it looks like if prefix is not defined, then we do not define the query as well, but still continue with processing.

I think we can exclude the entire if.